tests/unit/records/test_schemas_dc.py

```diff
@@ -184,3 +184,20 @@ def test_sources_meetings(minimal_record, recid_pid):
     }
     obj = dc_v1.transform_record(recid_pid, Record(minimal_record))
     assert obj['sources'] == ['CAP, Communicating, Cape Town, March, 2010']
+
+
+def test_description(minimal_record, recid_pid):
+    """Test description"""
+    minimal_record['descriptions'] = ['<p>TRPC1 and store-operated Ca2+&nbsp;(SOC) entry \
```

Comment:
Hey, the Travis is failing because the record model has a key "description" (singular) (see below: https://github.com/zenodo/zenodo/pull/1020/files#diff-d920dfe4ef6e66c934b0b9c68fc6ca57L112), while the exported DC format has a key "descriptions" (plural), hence you're not overwriting the initial field here. Simply change it to ``minimal_record["description"] = ...`` and it should work.

Also reduce this example to something simpler in text but more nested in terms of tags::

``'<p>Foo&nbsp;Bar baz&nbsp;<em><b><i>Spam</i></b></em>&nbsp;<i>Eggs</i></p>Foo``